### ISSUE TITLE
Messages do not arrive at the API when initiated by a new number 

### ISSUE THREAD

**Issue by AdrianoRobson** [pedroslopez/whatsapp-web.js#1716]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Messages do not arrive at the API when initiated by a number that has never sent a message to the number connected to the API before. Messages received by the API from a new number only appear from the second or third message sent by the new number

Messages are received by the API only after sending the second, third or fourth message sent by numbers that have never sent a message to the number connected to the API before

 
 
![bug6](https://user-images.githubusercontent.com/40671414/193854936-24c98b86-8be9-4424-98be-64b19a56ef25.png)

### Expected behavior

- A user who has never sent a message from a number to the number connected to the API starts a conversation by watsaap sending a message to a number connected to the API whatsaap-web.js
- The first message sent from this user number must be received by the API

### Steps to Reproduce the Bug or Issue

1 - Send a message from a whatsapp number that has never sent a message to the number connected to the API nor received a message from the number connected to the API

2 - See if the message arrived in the API

### Browser Type

Google Chrome

### WhatsApp Account Type

Standard

### Does your WhatsApp account have multidevice enabled?

Yes, I am using Multi Device

### Environment

OS: Ubuntu 22.04
whatsapp-web.js version v1.17.1
Node.js Version v16.13.1

**Comment by j-f-r** [comment]
Are you still facing this issue? I've found a similar issue in https://github.com/tulir/whatsmeow/issues/161

**Comment by krumpking** [comment]
Facing the same issue, my current workaround is to have a function to check for messages that were not replied to, but it is taxing

**Comment by AdrianoRobson** [comment]
> Are you still facing this issue? I've found a similar issue in [tulir/whatsmeow#161](https://github.com/tulir/whatsmeow/issues/161)

After running npm update whatsapp-web.js and carrying out some tests of sending message with new whatsapp numbers, apparently the problem stopped, however, I noticed that sometimes this problem happens again, there is no guarantee of receiving the message by api when a whatsapp number that has never sent or received a message to the number connected to the api sends the first message.

**Comment by j-f-r** [comment]
I'm having a different but maybe related problem.

Some of the messages I send (typically the first one in a span of time of like 1 day) shows as sent in the sender whatsapp, I receive all the ACK events, but do not show up in the recipient whatsapp. This is causing a lot of trouble for my clients.

Is anyone else facing this?

**Comment by alechkos** [comment]
#2292

**Comment by alechkos** [comment]
Duplicate of #1519
